python/py-fortranformat/Portfile

```diff
@@ -17,7 +17,7 @@ long_description    Generates text from a Python list of variables or will \
 checksums           md5 d78c5a320fedcbdf21f823cd18e28ac0 \
                     rmd160 212edf59a6da06b8127322f30806e6559353ea79 \
                     sha256 ec76c1c7bc07972aa98e01ff2303bb0aefe77306be8ff2723bc40b7c7775292c
-python.versions     27 34 35 36
+python.versions     27 35 36 37
```

Comment:
Please add the following code for the py34 port. 

```
subport py34-${python.rootname} {
    replaced_by py35-${python.rootname}
    PortGroup obsolete 1.0
}

```

Reply:
I am not sure about this; from the ticket I read that these replacement rules are kind of useless. I do not think we should be adding this code to every Portfile when removing a subport; if that is still needed I think we would be much better of keeping it centralized.

Reply:
Let me know which it is.

Reply:
It sounds like it is not needed.